From c673393591e280c7a0425ed8cdf560ecb4d65f64 Mon Sep 17 00:00:00 2001
From: Joao Rodrigues <joao.rodrigues@consensys.net>
Date: Thu, 13 Dec 2018 09:23:09 +0000
Subject: [PATCH] KOMGO-2729

---
 src/rabbit_mgmt_util.erl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rabbit_mgmt_util.erl b/src/rabbit_mgmt_util.erl
index 6235eeb1..98d163e7 100644
--- a/src/rabbit_mgmt_util.erl
+++ b/src/rabbit_mgmt_util.erl
@@ -711,7 +711,7 @@ id0(Key, ReqData) ->
     end.
 
 with_decode(Keys, ReqData, Context, Fun) ->
-    {ok, Body, ReqData1} = cowboy_req:read_body(ReqData),
+    {ok, Body, ReqData1} = cowboy_req:read_body(ReqData, #{length => infinity}),
     with_decode(Keys, Body, ReqData1, Context, Fun).
 
 with_decode(Keys, Body, ReqData, Context, Fun) ->
-- 
2.19.2

